index.html

```diff
@@ -30,8 +30,24 @@
     <article class="article-entry" id="links">
       <h2>Links will go here. Eventually</h2>
     </article>
+    <article class="article-entry" id="about">
+      <h2>About me: some games I like.</h2>
+      <section class="template">
```

Comment:
I believe we already touched on this, the class of template on the section element can be moved to the article element.